systemtests/tests/restore-test/functions

```diff
@@ -0,0 +1,19 @@
+runner_name=$(basename $0)
+log_home="$tmp/$runner_name"
+mkdir -p $log_home
+default_joblist="$log_home/jobs.out"
```

Comment:
double quote all to avoid shellcheck complaining.
```suggestion
runner_name="$(basename $0)"
log_home="${tmp}/${runner_name}"
mkdir -p "${log_home}"
default_joblist="${log_home}/jobs.out"
```

Reply:
Seems my editor did not recognize the file as a script file, so shell check was not run.

Reply:
My version of shellcheck seems to not like
```
runner_name="$(basename $0)"
```
and instead wants
```
runner_name=$(basename "$0")
```
Does that look ok to you and your version of shellcheck ?

Reply:
Mine indicate that without specific shell it doesn't know and if you indicate bash (which is used elsewhere see scripts/functions) then it want `runner_name="$(basename "$0")"`
I would go to shellcheck -s bash 

It will also recommend to use grep -c instead of grep ... | wc -l

Reply:
Mine also told me to change to grep -c.  Even with `-s bash` it does not want the " around the `basename` call. Maybe I have it setup incorrectly.  Since they will not break anything, I will just add them.